apps/src/templates/teacherDashboard/MultiSelectGroup.jsx

```diff
@@ -60,7 +60,15 @@ function MultiSelectButton({
   required,
   onCheckedChange
 }) {
-  const uniqueId = _.uniqueId();
+  const uniqueId = `multi-${_.uniqueId()}`;
```

Comment:
for whatever reason, firefox wasn't happy with these IDs starting with a number, even though that's supposed to be valid in html5. so i just switched it to start with a letter for better compatibility.